src/Compilers/CSharp/Test/Syntax/Parsing/ParsingTests.cs

```diff
@@ -319,5 +322,42 @@ private void Done(bool dump)
                 _output.WriteLine("EOF();");
             }
         }
+
+        protected static void ParseIncompleteSyntax(string text)
```

Comment:
the syntax tests are pretty fast, so I wonder if we'd benefit from making something like this happen automatically in UsingTree or similar methods in a future PR.

Reply:
The one test I added costs about 100ms. We probably have too many parsing tests to do this automatically. 
But I think it would be good for each feature to at least use it once.